### PR TITLE
Add data source edit tracking

### PR DESCRIPTION
**Pull request by jonpage**
Still needs testing

### CODE REVIEW

File: app/controllers/data_sources_controller.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ def clear_and_reload
   
   def delete
     source = DataSource.find_by id: params[:id]
+    create_action source,'DELETE'
```
Comment (kotodharma) [inline app/controllers/data_sources_controller.rb]: Might want to make this conditional on the success of `source.delete` (as you did for the update and create actions)

File: app/models/user.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 class User < ActiveRecord::Base
+  belongs_to :data_source_actions
```
Comment (kotodharma) [inline app/models/user.rb]: `has_many :data_source_actions`?

File: app/models/data_source_action.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class DataSourceAction < ActiveRecord::Base
+  belongs_to :series
+  has_one :users
```
Comment (kotodharma) [inline app/models/data_source_action.rb]: `belongs_to :users`? Or am I missing a subtlety of how we want the ORM stuff to work?

Comment (jonpage) [reply app/models/data_source_action.rb]: oops. thanks

**Pull request merged**